projectile.el

```diff
@@ -1415,7 +1415,11 @@ this case unignored files will be absent from FILES."
 (defun projectile-ignored-buffer-p (buffer)
   "Check if BUFFER should be ignored."
   (or
-   (member (buffer-name buffer) projectile-globally-ignored-buffers)
+   (cl-some
```

Comment:
OK, so this here is going to be a huge problem on bigger projects. Something that was very fast will become veeery slow. Keep in mind that you need to apply this functions to some pretty big lists of files here and there. People are already complaining that all the filtering makes projectile super slow and we're about to make it much slower.